index.html

```diff
@@ -245,19 +245,22 @@ <h1>The community I want isn't here!</h1>
             <h1>FAQ</h1>
             <h2>What is this?</h2>
             <p>
-                Codidact is a community initiative for developing a community-driven knowledge sharing website. We have some awesome ideas for that, but some communities
-                that are interested in using our software are in need of an immediate solution. Hence we used an existing Q&amp;A software. See our
+                Codidact is a community-driven, non-profit-focused network of knowledge sharing websites. Our motto is "For the community, by the community". See our
```

Comment:
I have a mostly irrational dislike of the word "motto"... can we say "our ethos is..." instead?

Reply:
reworded